spec/test_doubles/fake_class_double.rb

```diff
@@ -0,0 +1,9 @@
+class FakeClassDouble
```

Comment:
The name of the class is very mysterious... Can you think of a better one?

Reply:
Good point, I'll change this